PWGHF/D2H/Tasks/taskLcToK0sP.cxx

```diff
@@ -79,28 +217,73 @@ struct HfTaskLcToK0sP {
         continue;
       }
       */
+
       if (etaCandMax >= 0. && std::abs(candidate.eta()) > etaCandMax) {
         // Printf("Candidate: eta rejection: %g", candidate.eta());
         continue;
       }
 
       auto ptCand = candidate.pt();
-      registry.fill(HIST("hMassVsPtCand"), invMassLcToK0sP(candidate), ptCand);
       registry.fill(HIST("hPtCand"), ptCand);
+      registry.fill(HIST("hEtaCand"), candidate.eta());
+      registry.fill(HIST("hEtaCandVsPtCand"), candidate.eta(), ptCand);
+      registry.fill(HIST("hPhiCand"), candidate.phi());
+      registry.fill(HIST("hPhiCandVsPtCand"), candidate.phi(), ptCand);
+      registry.fill(HIST("hMass"), invMassLcToK0sP(candidate));
+      registry.fill(HIST("hMassVsPtCand"), invMassLcToK0sP(candidate), ptCand);
+      registry.fill(HIST("hPtBach"), candidate.ptProng0());
```

Comment:
Avoid repeated calls of the same functions with the same output. Store the values in variables and use them to fill the histograms.